src/Network/Network/help/New-AzPrivateEndpoint.md

```diff
@@ -24,14 +24,18 @@ The **New-AzPrivateEndpoint** cmdlet creates a private endpoint.
 ## EXAMPLES
 
 ### 1: Create a private endpoint
-```
+
+The following example creates a private endpoint with a specific private link service ID in the
+specified subnet in a virtual network.
+
```

Comment:
Maybe we should add a note regarding the values that needs to be adjusted according to the environment? For example "MyVirtualNetwork" and "TestResourceGroup"?

Reply:
You bring up a good point. I've removed the note about changing the value of 'mySubnet' since we're not specifically calling out the others. I've changed the values for each of the parameters to our standard ones. This should make it more cohesive, easier to transition from one doc to another, and for users to have a better idea of what needs to be changed if they've previously used any of the other Azure PowerShell related help docs.